app/account/account.rb

```diff
@@ -0,0 +1,27 @@
+class Account
```

Comment:
Please separate code to different entities like
account, account_manager, console, different card types etc.